bytecomp/translmod.ml

```diff
@@ -1038,8 +1055,28 @@ let transl_store_structure glob map prims str =
                  Lambda.subst no_env_update subst
                    (transl_module Tcoerce_none None modl),
                  store_idents 0 ids)
+        | Tstr_open od ->
+            begin match od.open_expr.mod_desc with
+            | Tmod_ident _ ->
+                transl_store rootpath subst rem
+            | _ ->
```

Comment:
It would be good to handle the `open struct ... end` case here specially. As the `include (struct ... end : sig ... end)` case is above.

Reply:
Done.